infra/main.tf

```diff
@@ -295,7 +298,7 @@ resource "time_sleep" "load_balancer_warm_up_time" {
     google_compute_global_forwarding_rule.http
   ]
 
-  create_duration = "370s"
+  create_duration = "150s"
```

Comment:
I really appreciate the use of data collection to identify a more optimal number; however, I have found these kinds of propagation delays can vary widely based on regional traffic, systems maintenance, and so on.

If this data was gathered during a single work session, it may be a best case scenario. Is this faster than product readiness guarantees? 

Instead of hard-coding this number, might we use https://registry.terraform.io/providers/hashicorp/http/latest/docs/data-sources/http with retries enabled (for 5xx responses) to detect when the forwarding rule is working as expected? (I'm not sure if 5xx errors are returned before it's ready)

Reply:
I hadn't considered that! Thanks, i'll take a look at that suggestion. Converting this PR to a draft because of course the int test failed at resolving the URL 👎🏽